driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -1075,16 +1105,16 @@ private String randomizePorts(CharSequence str) {
         }
 
         @Override
-        @SuppressWarnings("SimplifiableIfStatement")
         public boolean equals(Object o) {
-            // do not include cluster name and start, only
```

Comment:
Hrmm it doesn't look like cluster name is even part of the builder anymore since this is randomly generated, so maybe that comment isn't relevant anymore?